app/models/concerns/pageflow/hosted_file.rb

```diff
@@ -4,67 +4,50 @@ module HostedFile
     include UploadedFile
 
     included do
-      has_attached_file(:attachment_on_filesystem, Pageflow.config.paperclip_filesystem_default_options)
-      has_attached_file(:attachment_on_s3, Pageflow.config.paperclip_s3_default_options)
+      alias_attribute :file_name, :attachment_on_s3_file_name
```

Comment:
Let's put the `has_attached_file` call here instead of having each descendant define their own attachment. `HostedFile` can have a method that can be overridden to provide additional attachment options. Also, since there are only attachments on S3 now, I suggest renaming to simply `attachment`.